src/components/TodoWrapper.js

```diff
@@ -0,0 +1,40 @@
+import React, { useState } from 'react';
+import { TodoForm } from './TodoForm';
+import { Todo } from './Todo';
+import { EditTodoForm } from './EditTodoForm'; 
+import { v4 as uuidv4 } from 'uuid';
+uuidv4();
+
+export const TodoWrapper = () => {
+    const [todos, setTodos] = useState([]);
+
+    const addTodo = todo => {
+        setTodos([...todos, { id: uuidv4(), task: todo, completed: false, isEditing: false }]);
+        console.log(todos);
+    };
+    const toggleComplete = id => {
+        setTodos(todos.map(todo => todo.id === id ? { ...todo, completed: !todo.completed } : todo));
+    };
+    const deleteTodo = id => {
+        setTodos(todos.filter(todo => todo.id !== id));
+    };
+    const editTodo = id => {
+        setTodos(todos.map(todo => todo.id === id ? { ...todo, isEditing: !todo.isEditing } : todo));
+    };
+    const editTask = (task, id) => {
+        setTodos(todos.map(todo=> todo.id === id ? {...todo, task, isEditing: !todo.isEditing} : todo))
+    }
+    return (
+        <div className='TodoWrapper'>
+            <h1>To Do List</h1>
+            <TodoForm addTodo={addTodo} />
+            {todos.map((todo, index) => (
```

Comment:
you dont give a key if its EditTodoForm
also, its not recommanded to use key as index when the order of the items in the list can be changed, 
you have a uuid for each todo, use it